lib/jekyll-import/importers/wordpressdotcom.rb

```diff
@@ -22,16 +22,21 @@ def self.specify_options(c)
       end
 
       # Will modify post DOM tree
-      def self.download_images(title, post_hpricot, assets_folder)
+      def self.download_images(title, post_hpricot, assets_folder, published_at)
         images = (post_hpricot / "img")
         return if images.empty?
 
         Jekyll.logger.info "Downloading images for ", title
         images.each do |i|
           uri = i["src"]
 
-          i["src"] = format("{{ site.baseurl }}/%s/%s", assets_folder, File.basename(uri))
-          dst = File.join(assets_folder, File.basename(uri))
+          # Put the images into a /yyyy/mm/ subfolder to reduce clashes
+          datefolder = published_at.strftime("/%Y/%m")
+          assets_withdate_folder = format("%s%s", assets_folder, datefolder)
+          FileUtils.mkdir_p assets_withdate_folder
+
+          i["src"] = format("{{ site.baseurl }}/%s/%s", assets_withdate_folder, File.basename(uri))
+          dst = File.join(assets_withdate_folder, File.basename(uri))
```

Comment:
Wouldn't `File.join("{{ site.baseurl }}", dst)` return the same output as ln#38?

Reply:
Your right, that is WAY simpler.  I just slotted my extra variable into the existing code concat, but rewriting it is cleaner. Updated the PR with this